src/pages/Landing/index.test.tsx

```diff
@@ -7,7 +9,12 @@ import Landing from '.'
 jest.mock('hooks/useDisableNFTRoutes')
 
 describe('disable nft on landing page', () => {
+  beforeAll(() => {
+    store.dispatch(setOriginCountry('US'))
+  })
+
   it('renders nft information and card', () => {
+    store.dispatch(setOriginCountry('US'))
```

Comment:
Why do this again if already done in `beforeAll`?

Reply:
oops forgot to delete